webapp/src/components/add_issue/add_issue.jsx

```diff
@@ -44,10 +44,14 @@ export default class AddIssue extends React.PureComponent {
             attachToThread: false,
             previewMarkdown: false,
             assigneeModal: false,
+            isTyping: false,
         };
     }
 
     static getDerivedStateFromProps(props, state) {
+        if (state.isTyping) {
```

Comment:
every type stroke rerenders the component and causes messages to get the props value. This change tracks when the user is typing and will avoid this situation from happening.